modules/custom/az_media/config/install/field.storage.media.field_media_az_image.yml

```diff
@@ -13,16 +13,16 @@ field_name: field_media_az_image
 entity_type: media
 type: image
 settings:
+  target_type: file
```

Comment:
Delete this file's changes from this PR

- [ ] Create task to remedy config drift

Reply:
We should test running Troy's drush command on a clean copy of the main branch.  If this file gets updated there, we should omit this change from this PR and include it in a separate more generic config drift cleanup PR.

Reply:
@joeparsons This file was not included when running `lando drush az-core-distribution-config` on a clean install of Quickstart

Reply:
Ok, weird.  I guess we should keep these changes then.